src/Common/src/Interop/Windows/kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -68,10 +67,8 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint nFileSizeLow;
             internal uint dwReserved0;
             internal uint dwReserved1;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-            internal string cFileName;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 14)]
-            internal string cAlternateFileName;
+            internal FixedString.Size260 cFileName;
+            internal FixedString.Size14 cAlternateFileName;
```

Comment:
To associate functionality safely modifying those buffers. It's error-prone to access fixed arrays. Extension methods aren't workable, and you additionally need a way to know the actual size. If you have any alternative suggestions or improvements that provide safe access I'd love to hear them. It isn't pretty, but it was the best I could come up with after quite a bit of experimentation.

Note that there aren't many sizes of these fixed buffers in Windows APIs. I would expect no more than 7 or 8.